frontend/app/src/util/AppNavigation.ts

```diff
@@ -274,7 +274,8 @@ export class StrategyV2 {
 
         this.appNav.hostCommunicationMgr.sendMessageToHost({
           type: "SET_CURRENT_PAGE_NAME",
-          currentPageName: currentPageName,
+          // Make sure we don't send the official page name for the main page
```

Comment:
nit: could you please add a short sentence about the *why* we don't send it for the main page? it will help others in the future to refactor stuff to understand context around it